_rules/descriptive-link-5effbb.md

```diff
@@ -0,0 +1,292 @@
+---
+id: 5effbb
+name: Link in context is descriptive
+rule_type: atomic
+description: |
+  This rule checks that the accessible name of a link together with its context describe its purpose.
+accessibility_requirements:
+  wcag20:2.4.4: # Link Purpose (In Context)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
```

Comment:
I'm wondering if this rule should also map to 2.4.9 It is not clear what we should do in such case (one SC is a subset of another).
More discussion on the topic: https://github.com/act-rules/act-rules.github.io/pull/833#discussion_r351650002

Reply:
I see how this works for fail examples of 2.4.4 (they also fail 2.4.9). But what about passing examples of 2.4.4 that fail 2.4.9? It would require a manual interpretation of the result of the rule if you are looking at SC 2.4.9 specifically.
Having it as two rules, each dedicated to a specific SC solves that "ambiguity"

Reply:
As usual, the important is the fail-on-fail relationship.
- Failing this rule fails both 2.4.4 and 2.4.9
- Passing this rule requires further testing for 2.4.9 (and passes 2.4.4 if we assume that all links have a role of `link`).

This is perfectly fine in term of rule design and this is what happen with most rules (passing the rule requires further testing).

I agree that this does collide with our goal that "passed examples must pass the SC"… 🤔

Reply:
I think this rule needs the mapping to 2.4.9
Failing this rule also fails 2.4.9 (if the link in context is not descriptive, then the link alone is not descriptive); this is a consequence of "failing 2.4.4 also fails 2.4.9 which is more precise".

As per [ACT Rules Format](https://www.w3.org/TR/act-rules-format/#accessibility-requirements-mapping):
> the rule must list all accessibility requirements from those documents that are not satisfied when one or more of the outcomes of the rule is failed. 

So, the rule **must** list both 2.4.4 and 2.4.9 as accessibility requirements.

Reply:
Posted this in the contrast rule too. I don't think we need to worry about the passed examples passing 2.4.9 here, as long as they all pass 2.4.4. I don't think there's a way around it for these rules where the level AAA one is a subset of the level A variant.